dashboard/app/models/user.rb

```diff
@@ -1063,6 +1063,7 @@ def recent_courses_and_scripts
 
     course_data = courses.map do |course|
       {
+        id: course[:name],
```

Comment:
Feels kind of strange that `recent_course[:id]` is the same thing as `course[:name]` and `recent_course[:name]` is something different. Should this instead be:
```
name: course[:name],
title: data_t_suffix('course.name', course[:name], 'title'),
```